python/pyarrow/_flight.pyx

```diff
@@ -1150,6 +1156,38 @@ cdef class FlightClient(_Weakrefable):
                 self.client.get().Authenticate(deref(c_options),
                                                move(handler)))
 
+    def authenticateBasicToken(self, username, password,
```

Comment:
Let's follow Python naming conventions - this should use snake_case.

Reply:
Addressed.